src/init/commands/index.test.ts

```diff
@@ -0,0 +1,102 @@
+import { afterEach, describe, expect, it, vi } from "vitest";
+import inquire from "inquirer";
+import { Command } from "commander";
+import { installInitCommands, makeOptions } from "./index.js";
+import { initAction } from "../initAction.js";
+
+vi.mock("../initAction.js");
+vi.mock("inquirer");
+
+const TEST_OPTIONS = {
+  channelId: "123",
+  name: "My App",
+  viewType: "compact",
+  endpointUrl: "https://example.com",
+};
+
+describe("makeOptions", () => {
+  // 1
+  it("should return parameters as is when all parameters are given", async () => {
+    // prompt input will be ignored
+    vi.mocked(inquire.prompt).mockResolvedValue({
+      channelId: "789",
+      name: "hoge",
+      viewType: "tall",
+      endpointUrl: "https://this-will-be-ignored.com",
```

Comment:
```suggestion
      endpointUrl: "https://example.com/this-will-be-ignored",
```

It would be better to use a single dummy domain for the test cases.
If we use other domains, we don’t know when or by whom they might be acquired.